README.md

```diff
@@ -1,27 +1,89 @@
 # @typescript/analyze-trace
-Tool for analyzing the output of `tsc --generateTrace` automatically, rather than following the steps [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
+
+A tool for analyzing the output of `tsc --generateTrace` in a fast and digestible way (rather than more involved diagnostics [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
 
 Note: The goal is to identify clear-cut hot-spots and provide enough context to extract a small repro.
 The repro can then be used as the basis of a bug report or a starting point for manual code inspection or profiling.
 
 ## Usage
 
-First, build your project with `tsc --generateTrace traceDir`.  This will create a new `traceDir` directory with paired trace and types files.
+The short version is to run these commands:
+
+```sh
+tsc -p tsconfig.json --generateTrace traceDir
+npm install --no-save @typescript/analyze-trace
+npx analyze-trace
+```
+
+Each of these commands do the following:
+
+1. First you will need to build your project with `--generateTrace` to a specific directory.
+   `tsc -p tsconfig.json --generateTrace traceDir` will create a new `traceDir` directory with paired trace and types files.
+   If your configuration file is not `tsconfig.json`, you will need to adjust the command.
+2. Installing `@typescript/analyze-trace` makes its various commands available in your project.
+3. Running `npx analyze-trace traceDir` outputs a sorted list of compilation hot-spots - places where TypeScript is taking a high amount of time.
 
-Next, install `@typescript/analyze-trace` to make its various commands available in your project.
+For best results, the analyzer should run on trace files that were generated in the same relative location.
+If the trace files fall out of date with your project, you may see different results.
+https://github.com/microsoft/typescript-analyze-trace/issues/29
+You can run `npx analyze-trace --help` to find out about other options including:
 
-For a sorted list of compilation hot-spots, run `npx analyze-trace traceDir`.
-Pass `--help` to learn more about configuration options.
-For best results, run the analyzer on a machine where the paths in the trace file(s) resolve correctly.
+Option                    | Default | Description
+--------------------------|---------|-------------------------------------------------------------------------------
+`--skipMillis [number]`   | `100`   | Suppress events that take less than the specified number of milliseocnds
```

Comment:
I actually missed out on `forceMillis`

Reply:
`skipMillis` always takes precedence. Otherwise, items are only displayed if they exceed `forceMillis` or they take a surprising percentage (currently 60%) of their parent task's time.

I would guess `skipMillis` is usually the thing you want to increase if you need to hunt down the biggest offenders first.

One you're out of those and you want to find more candidates, you would drop `skipMillis` and *reduce* `forceMillis` when you want to find chunks of work that take a certain amount of time, or to dive deeper into a tree to find the root cause.

Maybe we should have a section on that.